dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -138,12 +130,12 @@ public Result createProcessDefinition(@ApiIgnore @RequestAttribute(value = Const
      * @param targetProjectCode target project code
      * @return copy result code
      */
-    @ApiOperation(value = "copy", notes = "COPY_PROCESS_DEFINITION_NOTES")
+    @ApiOperation(value = "batchCopyProcessDefinitions", notes = "COPY_PROCESS_DEFINITION_NOTES")
```

Comment:
please check

Reply:
I have put it another way.
`@ApiOperation(value = "batchMoveByCodes", notes = "MOVE_PROCESS_DEFINITION_NOTES")`